projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -17,19 +17,36 @@
 	background-position: center;
 }
 
-.jp-carousel-overlay .swiper-button-prev:after,
-.jp-carousel-overlay .swiper-button-next:after {
-	color: #cccccc;
+.jp-carousel-overlay .swiper-button-prev,
+.jp-carousel-overlay .swiper-button-next {
 	opacity: 0.5;
-	font-size: 3rem;
 	transition: 0.5s opacity ease-out;
+	height: initial;
+	width: initial;
 }
 
-.jp-carousel-overlay .swiper-button-prev:hover:after,
-.jp-carousel-overlay .swiper-button-next:hover:after {
+.jp-carousel-overlay .swiper-button-prev:hover,
+.jp-carousel-overlay .swiper-button-next:hover {
 	opacity: 1;
 }
 
+.jp-carousel-overlay .swiper-button-prev:after,
+.jp-carousel-overlay .swiper-button-next:after {
+	content: none;
+}
+
+.jp-carousel-overlay .swiper-button-prev svg,
+.jp-carousel-overlay .swiper-button-next svg {
+	height: 40px;
+	width: 40px;
+	fill: #ccc;
+}
+
+.jp-carousel-overlay .swiper-button-prev svg:hover,
+.jp-carousel-overlay .swiper-button-next svg:hover {
+	fill: #fff;
```

Comment:
I think we might need to include a light mode variant for this rule, otherwise when we hover over the button it blends into the background. In the following screenshot I'm hovering over the next button:

![image](https://user-images.githubusercontent.com/14988353/123892863-b5050f80-d99e-11eb-8944-74be3c4e7084.png)